Operational-Procedures/Scheduler/night-time-operations/index.rst

```diff
@@ -0,0 +1,255 @@
+This is a special configuration done on the *scheduling algorithm* side alonside the survey operation.
+
+Nevertheless, because at the beginning of the night the optics may be far from the optimum position, it is recommended that a initial alignement is done before starting the Scheduler.
+
+The best way to accomplish this is to execute the same SAL Script used by the Scheduler on the ScriptQueue.
+For the AT, this is ``auxtel/latiss_cwfs_align.py``.
+This SAL Script accepts as inputs an az/el position on the sky and the magnitude limit for a source finding algorithm.
+It will then, query Simbad to find a suitable target around the specified position with the specified magnitude limit and perform wavefront sensing.
+An example configuration is as follows:
+
+.. code-block:: text
+
+    find_target:
+      az: 100.0
+      el: 60.0
+      mag_limit: 8
+
+.. _scheduler-night-time-operation-starting-scheduler:
+
+Starting the Scheduler
+======================
+
+After the initial optical alignment is completed successfully. it is safe to start the Scheduler.
+For that, we simply need to send the ``resume`` command to the CSC.
+This command has no parameters and can be executed directly from the ScriptQueue, using the ``run_command`` script with the following configuration:
+
+.. code-block:: text
+
+    component: Scheduler:2
+    cmd: resume
+
+.. figure:: ./_static/atqueue-run-cmd-scheduler-resume.png
+    :name: fig-atqueue-run-cmd-scheduler-resume
+
+    Starting the AT Scheduler from the ATQueue view on LOVE
+
+or from nublado:
+
+.. code:: ipython3
+
+    await remote.cmd_resume.set_start(timeout=5)
+
+Once the Scheduler CSC starts, it will gather talemetry and compute a set of targets to send to the ScriptQueue.
+At this point it is important to keep and eye on the Scheduler logging output, to follow up the different steps it takes during the process.
+
+For the Auxiliary Telescope, the `AT Summary State`_ view provides a good overview of the system.
+A screen shot of this view, taken whith most of the AT components in ``STANDBY`` state is shown :ref:`below <fig-at-summary-state-view>`.
+
+It is also highly recommended to have the `ATQueue`_ view opened at all times.
+The Scheduler will queue 2 observations ahead of time, so you should expect to have one observation executing and another 2 waiting to be executed.
+As the scripts finishes executing, the Scheduler will continue to add targets to the queue.
+
+.. _AT Summary State: http://amor01.cp.lsst.org/uif/view?id=62
+.. _ATQueue: http://amor01.cp.lsst.org/uif/view?id=41
+
+.. figure:: ./_static/at-summary-state-view.png
+    :name: fig-at-summary-state-view
+
+    AT Summary State view from LOVE at the summit.
+    At the top the view contains the summary state of the most relevant components. 
+    At the bottom, on the right-hand side, is shown the dome and telescope position and on the left-hand side the logging from the Scheduler.
```

Comment:
Consider:
On the bottom right, the dome and telescope position are shown.
The left-hand side shows the logging output from the Scheduler.